src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
you need to define certain strings for this, create two uniq type Color and Shape

Reply:
Hi. Ok, I`ve done create two unique type Color and Shape.